python/dataStructure_process.py

```diff
@@ -0,0 +1,12 @@
+'''
+    TRAIN TEST SPLIT
+    Split the dataset in two according to a specified proportion
+    output : 
+        train, test : two datasets
+'''
+def train_test_split(data, training_proportion):
```

Comment:
Same comment as in the last pull request, you might use https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html or add random seed and shuffle capability.

Reply:
Yes, thanks!
For now, we use this really basic version as we are still trying to find a way (like using hash of the train/test data?) to be certain that several models have been trained and tested on the same data sets, in case someone imports a trained model and want to compare its performance to a new trained model.